bemb/model/bemb.py

```diff
@@ -1298,7 +1316,17 @@ def elbo(self, batch: ChoiceDataset, num_seeds: int = 1) -> torch.Tensor:
         # ==============================================================================================================
         # 1. sample latent variables from their variational distributions.
         # ==============================================================================================================
-        sample_dict = self.sample_coefficient_dictionary(num_seeds)
+        if self.deterministic_variational:
+            num_seeds = 1
```

Comment:
Do we really need the `num_seeds` variable here?

Reply:
no, we do not; removing.